src/ncp/ncp_base.hpp

```diff
@@ -575,10 +579,12 @@ class NcpBase
 #endif
     ThreadError InsertPropertyHandler_IPV6_ADDRESS_TABLE(uint8_t header, spinel_prop_key_t key,
                                                          const uint8_t *value_ptr, uint16_t value_len);
+#if OPENTHREAD_FTD
     ThreadError InsertPropertyHandler_THREAD_LOCAL_ROUTES(uint8_t header, spinel_prop_key_t key,
```

Comment:
Agree.  The Thread Spec does not preclude an MTD from implementing border router functionality.  I think we should have a separate "OPENTHREAD_ENABLE_BORDER_ROUTER" feature flag that wraps border router related code.